### PR TITLE
Refactor string literals continued

### PR DESCRIPTION
**Pull request by mrinsin**

### CODE REVIEW

File: web-client/src/presenter/computeds/caseTypeDescriptionHelper.js [hunk 1]
```diff
@@ -70,29 +70,29 @@ export const caseTypeDescriptionHelper = (get, applicationContext) => {
...
-        case 'Whistleblower':
+        case CASE_TYPES_MAP.whistleblower:
           caseDescription = 'Whistleblower';
           break;
-        case 'Worker Classification':
+        case CASE_TYPES_MAP.workerClassification:
           caseDescription = 'Worker Classification';
           break;
-        case 'Declaratory Judgment (Retirement Plan)':
+        case CASE_TYPES_MAP.djRetirementPlan:
```
Comment (matthopson) [inline web-client/src/presenter/computeds/caseTypeDescriptionHelper.js:85]: D5zH9SyxCKdC62Ei6Uwo5CyLmWyrWSsLNte9qLuRBaHNo4M62yTvdhKqeTZqpjr1ZLg3QmzZYsEiFEDVN1SDF2x69rmycCQpDQUs8GavpqM7ugdXiLA44QDRDY7JpN1U6oFuEu

**Review by matthopson — APPROVED** [review]
I noticed in a previous PR, caps were used on the object values like: `CONSTANTS_OBJECT.VALUE` - where this is `CONSTANTS_OBJECT.value`

Curious what steered those

**Review by ericsorenson — APPROVED** [review]

**Comment by mrinsin** [comment]
@matthopson I noticed that too... not sure why different constant objects were written differently on creation but I am all for standardizing the constants in that file. I could make a devex task out of that :)

**Pull request merged**